### DISCUSSION TITLE
Trial?

### DISCUSSION THREAD

**Discussion by JPT62089** [rustdesk/rustdesk-server-pro#44] [General]
Hello! Is it possible to set up a trial for Pro? I would love to test it out and see how it would work for me

**Comment by dinger1986** [comment]
Currently no, if you have tested the public rustdesk it's basically extra features ontop 😀

**Comment by JPT62089** [comment]
Yup, totally get that but I would also like to try those extra features out to see if it will work for our company and our use-case. The web console is specifically what I am wanting to test out :)
